plugins/modules/one_vm.py

```diff
@@ -157,7 +157,7 @@
     type: float
   vcpu:
     description:
-      - Number of CPUs (cores) new VM will have.
+      - Number of CPUs (cores) new VM is set to have.
```

Comment:
Hmm, 'is set to' sounds pretty strange here. How about either keeping 'will', or writing something like:
```suggestion
      - Number of CPUs (cores) the new VM uses.
```